From 3471ed0e244869ba097478f01a9b6aed812164d0 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 24 May 2020 19:50:41 +0300
Subject: [PATCH] [SW-9973] gaudi: finally enable hpm

Current protection mask lacks the Status Register Write Protection
(SRWD) bit and therefore does not enable the Hardware Protection Mode
(HPM).
Add the SRWD bit to the Gaudi protection mask.

Change-Id: I5a84c37ef225fb2ed05e14f9a0b352736ae37ba7
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/common.h b/gaudi/common.h
index 4869859..046ec26 100644
--- a/gaudi/common.h
+++ b/gaudi/common.h
@@ -47,7 +47,7 @@
 #undef PROT_MASK
 #endif
 /* Only BP2 == 0x4 - means 8 blocks - up to 512KB */
-#define PROT_MASK			READ_STATUS_BP2
+#define PROT_MASK			(READ_STATUS_BP2 | READ_STATUS_SRWD)
 
 #endif /* __ASSEMBLY__ */
 #endif /* COMMON_GAUDI_H_ */
